src/pages/__tests__/get.test.js

```diff
@@ -1,138 +1,40 @@
-    expect.assertions(1)
-    getBrowserInfo.mockReturnValue(createMockBrowserInfo('firefox', false))
-    const GetExtensionRedirectPage = require('../get').default
-    const mockProps = getMockProps()
-    shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(firefoxExtensionURL)
-  })
-
-  it('redirects to the Firefox Add-ons page when the browser is Firefox on mobile', () => {
-    expect.assertions(1)
-    getBrowserInfo.mockReturnValue(createMockBrowserInfo('firefox', true))
-    const GetExtensionRedirectPage = require('../get').default
-    const mockProps = getMockProps()
-    shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(firefoxExtensionURL)
-  })
-
-  it('redirects to the Edge Add-ons page when the browser is Edge on desktop', () => {
-    expect.assertions(1)
-    getBrowserInfo.mockReturnValue(createMockBrowserInfo('edge', false))
-    const GetExtensionRedirectPage = require('../get').default
-    const mockProps = getMockProps()
-    shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(edgeExtensionURL)
-  })
-
-  it('redirects to the Edge Add-ons page when the browser is Edge on mobile', () => {
-    expect.assertions(1)
-    getBrowserInfo.mockReturnValue(createMockBrowserInfo('edge', true))
     const GetExtensionRedirectPage = require('../get').default
     const mockProps = getMockProps()
     shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(edgeExtensionURL)
   })
 
-  it('redirects to the homepage when it is an unsupported browser', () => {
-    expect.assertions(1)
-    getBrowserInfo.mockReturnValue(createMockBrowserInfo('safari', false))
-    const GetExtensionRedirectPage = require('../get').default
-    const mockProps = getMockProps()
-    shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(homeURL)
-  })
-
-  it('redirects to the homepage when the browser value is null', () => {
-    expect.assertions(1)
-    getBrowserInfo.mockReturnValue(createMockBrowserInfo(null, false))
-    const GetExtensionRedirectPage = require('../get').default
-    const mockProps = getMockProps()
-    shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(homeURL)
-  })
-
-  it('redirects to the homepage if browser detection throws an error', () => {
-    expect.assertions(1)
-    getBrowserInfo.mockImplementation(() => {
-      throw new Error('I could not browse browser :(')
-    })
+  it('redirects to the proper browser extension', () => {
     const GetExtensionRedirectPage = require('../get').default
     const mockProps = getMockProps()
     shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(homeURL)
+    expect(redirect).toHaveBeenCalled()
+    expect(directToAppExtension).toHaveBeenCalled()
```

Comment:
A better test: mock a return URL for `directToAppExtension`, then assert redirect is called with that URL.